scalableminds/webknossos
# Fix volume version restore for untouched buckets

`VolumeTracingService.revertToVolumeVersion` checked against “Empty” to select which buckets need to be restored to empty. However, the fossildb client gave Failure instead of Empty for missing buckets. This is why the version restore skipped those buckets.

### URL of deployed dev instance (used for testing):
- https://volumerestore.webknossos.xyz

### Steps to test:
- Create volume annotation, brush some, save, then restore version 0, reload page, should still look like version 0

### Issues:
- fixes #5717 

------
